python/rmm/mr.py

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.
+
+from rmm._lib.memory_resource import (  # noqa: F401
```

Comment:
If they're defined in `__all__` do we still need the noqa here?

Reply:
The `noqa` sliences the flake8 errors relating to unused imports.

The `__all__` is for ensuring that `help(rmm.mr)` includes documentation for all the types imported here.

IIUC, since `__all__` is just a list of strings, the noqa is still required

Reply:
I could have sworn the `__all__` fixes the flake8 error, is this not the case?

Reply:
done